[2021-08-04 14:32:43,480] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-08-04T12:32:39.795977+00:00 [queued]>
[2021-08-04 14:32:43,485] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: trigger_rule.task_3 2021-08-04T12:32:39.795977+00:00 [queued]>
[2021-08-04 14:32:43,485] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:32:43,485] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 14:32:43,485] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 14:32:43,493] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): task_3> on 2021-08-04T12:32:39.795977+00:00
[2021-08-04 14:32:43,495] {standard_task_runner.py:52} INFO - Started process 1347539 to run task
[2021-08-04 14:32:43,499] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'trigger_rule', 'task_3', '2021-08-04T12:32:39.795977+00:00', '--job-id', '22', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/trigger_dag.py', '--cfg-path', '/tmp/tmp03l3s4k9', '--error-file', '/tmp/tmpccccgua5']
[2021-08-04 14:32:43,499] {standard_task_runner.py:77} INFO - Job 22: Subtask task_3
[2021-08-04 14:32:43,522] {logging_mixin.py:104} INFO - Running <TaskInstance: trigger_rule.task_3 2021-08-04T12:32:39.795977+00:00 [running]> on host adam
[2021-08-04 14:32:43,547] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=trigger_rule
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T12:32:39.795977+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-04T12:32:39.795977+00:00
[2021-08-04 14:32:43,548] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-04 14:32:43,548] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'exit 0']
[2021-08-04 14:32:43,553] {subprocess.py:74} INFO - Output:
[2021-08-04 14:32:43,554] {subprocess.py:82} INFO - Command exited with return code 0
[2021-08-04 14:32:43,561] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=trigger_rule, task_id=task_3, execution_date=20210804T123239, start_date=20210804T123243, end_date=20210804T123243
[2021-08-04 14:32:43,574] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-04 14:32:43,590] {local_task_job.py:149} INFO - Task exited with return code 0
